src/main/scala/circt/stage/ChiselStage.scala

```diff
@@ -50,6 +47,14 @@ object ChiselStage {
   /** Elaborate a Chisel circuit into a CHIRRTL string */
   def emitCHIRRTL(gen: => RawModule): String = chisel3.stage.ChiselStage.emitChirrtl(gen)
 
+  /** Return a CHIRRTL circuit for a Chisel module
+    *
+    * @param gen a call-by-name Chisel module
+    */
+  def convert(gen: => RawModule): firrtl.ir.Circuit = {
+    chisel3.stage.ChiselStage.convert(gen)
+  }
```

Comment:
Nit: It may be better to copy the body of the method than to have `circt.stage.ChiselStage` call `chisel3.stage.ChiselStage`.